_people/LionelRobert.md

```diff
@@ -0,0 +1,13 @@
+---
+first_name: Lionel P.
+last_name: Robert Jr.
+title: Associate Professor of Information, School of Information and director of the Michigan Autonomous Vehicle Research Intergroup Collaboration
```

Comment:
Leave the lab group to the bio section, so just Associate Professor of Information

Reply:
Made edits to address both